### PR TITLE
docs/Fix overview section

### PR DESCRIPTION
**Pull request by vifer**

### CODE REVIEW

File: addresses.go [hunk 1]
```diff
@@ -1,4 +1,9 @@
+//
```
[the remark this answers is not in the document]
Comment (alecsammon) [reply addresses.go:1]: actually I think the only thing that is needed is the line break between the comment and the `package paddle`

[the remark this answers is not in the document]
Comment (alecsammon) [reply addresses.go:1]: having a single line break seams to be what `mockery` does

[the remark this answers is not in the document]
Comment (mikeymike) [reply addresses.go:1]: ^ can confirm a line break does seem to do the trick

[the remark this answers is not in the document]
Comment (vifer) [reply addresses.go:1]: done ✅

**Review by alecsammon — APPROVED** [review]

**Review by mikeymike — APPROVED** [review]

**Pull request merged**